templates/first-snap/_layout_fsf.html

```diff
@@ -10,11 +10,9 @@
   </div>
 
   {% if self.fsf_pagination() %}
-  <div class="p-strip u-no-padding snapcraft-banner-background">
+  <div class="p-strip u-no-padding">
```

Comment:
If pagination now doesn't need snapcraft background I guess it doesn't need to be in own strip anymore?
Maybe moving it together with content into one strip would allow to get rid of some `u-no-padding` workarounds?